wweval/R/get_input_data.R

```diff
@@ -43,51 +48,67 @@ get_input_ww_data <- function(forecast_date_i,
   ww_data_path <- file.path(ww_data_dir, paste0(date_to_pull, ".csv"))
   raw_nwss_data <- readr::read_csv(ww_data_path, show_col_types = FALSE)
 
-  # Use package functions to subset NWSS data
-  ww_data <- raw_nwss_data |>
-    init_subset_nwss_data()
+  # Use wweval functions to subset NWSS data and format for wwinference package
+  all_ww_data <- raw_nwss_data |>
+    clean_and_filter_nwss_data()
   # Get the data corresponding to the scenario
   list_of_site_ids <- get_scenario_site_ids(
-    ww_data,
+    all_ww_data,
     scenario_i,
     scenario_dir
   )
-  subsetted_ww_data <- ww_data |>
+  subsetted_ww_data <- all_ww_data |>
     dplyr::filter(wwtp_name %in% !!list_of_site_ids)
 
-  ww <- subsetted_ww_data |>
+  ww_data_pkg <- subsetted_ww_data |>
```

Comment:
I'm guessing this rename to `ww_data_pkg` was to mark it as part of the transition to using `wwinference`. I think we can move back to calling this something like `cleaned_ww_data`. I also think this pipe chain can be longer; not clear to me that `subsetted_ww_data` and `ww_data_pkg` need to be defined as separate variables on the way to `deduplicated_data`